src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqBwaSpark.java

```diff
@@ -152,12 +153,12 @@
                                                              final ReadsSparkSource readsSource) {
         if (path == null) return null;
```

Comment:
Can these paths be input as Path specifiers instead of strings here?

Reply:
Yes, but it pulls on a longer thread. The pathseq package needs its own PR.